server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -18,52 +18,37 @@
  */
```

Comment:
Just a question here, `ensureStorageGroup` use the default level as the storage group level. If the user already set some other storage group whose level is not equal to the default level, what will happen ?

Reply:
StorageGroupAlreadSetException will be thrown.